src/worker.js

```diff
@@ -133,6 +133,7 @@ class Worker {
     return Promise.fromCallback((cb) => {
       cls.run(() => {
         cls.set('tid', this.tid)
+        cls.set('previousEvent', this.queue)
```

Comment:
thinking harder about it in this context I dont think  `previousEvent` is the right word here. It seems more like `currentWorkerName` would be better here?